support-frontend/assets/pages/weekly-subscription-checkout/components/weeklyCheckoutForm.tsx

```diff
@@ -193,20 +193,25 @@ function WeeklyCheckoutForm(props: PropTypes) {
 		);
 	});
 
+	const hideSummaryImage =
+		props.participations.subscribeCheckoutImage === 'variant';
+
 	return (
 		<Content>
 			<Layout
 				aside={
 					<>
 						<Summary
 							image={
-								<GridImage
-									gridId="checkoutPackshotWeekly"
-									srcSizes={[500]}
-									sizes="(max-width: 740px) 50vw, 548"
-									imgType="png"
-									altText=""
-								/>
+								!hideSummaryImage ? (
```

Comment:
I wonder if it'd make it slightly easier to read this if some of the negation was removed? For example above we could do:

```
const showSummaryImage =
		props.participations.subscribeCheckoutImage !== 'variant';
```

And then this line could be:

```
showSummaryImage ? (
```

What do you reckon? Obviously very minor so no worries either way!

Reply:
Yes, agree, not great to have a double negative unnecessarily, changed